pkg/scaffold/resource/resource.go

```diff
@@ -16,144 +16,42 @@ limitations under the License.
 
 package resource
 
-import (
-	"fmt"
-	"regexp"
-	"strings"
-
-	"github.com/gobuffalo/flect"
-)
-
 // Resource contains the information required to scaffold files for a resource.
 type Resource struct {
-	// Namespaced is true if the resource is namespaced
-	Namespaced bool
+	// Namespaced is true if the resource is namespaced.
+	Namespaced bool `json:"namespaces,omitempty"`
 
-	// Group is the API Group.  Does not contain the domain.
-	Group string
+	// Group is the API Group. Does not contain the domain.
+	Group string `json:"group,omitempty"`
 
-	// GroupImportSafe is the API Group.  Does not contain the domain and it the "-"
-	// It is used to do safe imports.
-	GroupImportSafe string
+	// GroupPackageName is the API Group cleaned to be used as the package name.
+	GroupPackageName string `json:"-"`
```

Comment:
`json:"-"`? Why ?

Reply:
`GroupPackageName` and `ImportAlias` are used for scaffolding purposes but can be easily obtained from the rest of the fields. Serializing them to json/yaml doesn't make sense, they would be duplicated information.